Source/Filter/Filter.cpp

```diff
@@ -153,6 +155,23 @@ void Filter::DoFilter (const Box& tbx,
             }
         }
 
+        dst(i,j,k,dcomp+n) = d;
+    });
+#else
+    AMREX_PARALLEL_FOR_4D ( tbx, ncomp, i, j, k, n,
+    {
+        Real d = 0.0;
+
+        for         (int iz=0; iz < slen_local.z; ++iz){
+            for     (int iy=0; iy < slen_local.y; ++iy){
+                for (int ix=0; ix < slen_local.x; ++ix){
+                    Real sss = sx[ix]*sz[iy];
+                    d += sss*( tmp(i-ix,j,k,scomp+n)
+                              +tmp(i+ix,j,k,scomp+n));
```

Comment:
I think that the code in 2D and 3D changed recently. Therefore, it should be:
```suggestion
                    Real sss = sz[iy];
                    d += sss*( src_zeropad(i-ix,j,k,scomp+n)
                              +src_zeropad(i+ix,j,k,scomp+n));
```

Reply:
Got it. Done!